balancer-js/src/modules/pools/pool-types/concerns/composableStable/exit.concern.ts

```diff
@@ -16,20 +83,68 @@ export class ComposableStablePoolExit implements ExitConcern {
     wrappedNativeAsset,
     singleTokenMaxOut,
   }: ExitExactBPTInParameters): ExitExactBPTInAttributes => {
-    // TODO implementation
-    console.log(
-      exiter,
-      pool,
+    this.checkInputsExactBptIn(
       bptIn,
-      slippage,
-      shouldUnwrapNativeAsset,
-      wrappedNativeAsset,
-      singleTokenMaxOut
+      singleTokenMaxOut,
+      pool,
+      shouldUnwrapNativeAsset
     );
 
-    throw new Error('To be implemented');
-  };
+    const sortedValues = this.getValuesSorted({
```

Comment:
Lets keep consistency and follow the same pattern as the join concern. So call it `sortValuesBasedOnPoolVersion`.